_posts/2020-03-28-learning-common-keyboard-shortuts-for-screenreaders.md

```diff
@@ -46,7 +46,7 @@ The presenter demonstrates the common hand movements to bring up the rotor menu,
 ## Learning with a quiz
 Other individuals may learn better testing their memory through a quiz. If creating a quiz may be too time consuming, there is a bookmark available below.
 
-Bookmarklet: [Quiz for Keyboard shortcuts on Deque](javascript:(()%20=>%20%7B%20$('.data.shortcuts tr').each((i,v)%20=>%20%7B%20const inputLabel%20=%20$('%5Bscope="row"%5D',v).text();%20const%20$input%20=%20`<input aria-label="$%7BinputLabel%7D" type="text" />`;%20$('td:last-child',v).html($input);%20%7D)%7D)();) (drag to bookmarklets bar or right click and save as bookmark).
+Bookmarklet: [Quiz for Keyboard shortcuts on Deque](javascript:%28%28%29%20=>%20%7B%20$%28'.data.shortcuts tr'%29.each%28%28i,v%29%20=>%20%7B%20const%20inputLabel%20=%20$%28'%5Bscope="row"%5D',v%29.text%28%29;%20const%20$input%20=%20`<input aria-label="$%7BinputLabel%7D" type="text" />`;%20$%28'td:last-child',v%29.html%28$input%29;%20%7D%29%7D%29%28%29;) (drag to bookmarklets bar or right click and save as bookmark).
```

Comment:
Hi @scottaohara I'm trying to add a link to a bookmarklet and I'm not sure how to do so with Markdown.
The original code is on my gist: https://gist.github.com/anevins12/de11bd407f7d63e8b121e49991ae6385

I've been using someone's readme file as an example because they've been able to achieve it: 
* https://github.com/retrography/VUTt/blob/master/README.md ("drag the following link")
* Raw: https://raw.githubusercontent.com/retrography/VUTt/master/README.md

Reply:
I've removed the bookmark and linked to CodePen instead